### PR TITLE
Update CSS token per new design system theming capability

### PR DESCRIPTION
**Pull request by fajar-apri-alaska**
# Alaska Airlines Pull Request

**Fixes:** #269 

## Summary:

- Update the design tokens to reflect the new theming conventions regarding tokens from the AuroDesignTokens and JetstreamDesignTokens repo.

## Type of change:

Please delete options that are not relevant.

- [x] Revision of an existing capability

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: src/style.scss [hunk 1]
```diff
@@ -363,21 +366,22 @@ input {
...
 
       @include auro_breakpoint--sm {
-        box-shadow: inset 0 0 0 1px var(--auro-color-ui-default-on-light);
+        box-shadow: inset 0 0 0 1px var(--ds-color-ui-default-default, $ds-color-ui-default-default);
       }
     }
 
     &:disabled {
+      // where does this "auro-color-alert-disabled-on-light" refers to?
       border-color: var(--auro-color-alert-disabled-on-light);
```
Comment (fajar-apri-alaska) [inline src/style.scss:385]: Hi @jason-capsule42, need advise on this token value.

Comment (jason-capsule42) [reply src/style.scss:385]: There is no disabled version of this. Looks like there was a bug and it should have been `--auro-color-alert-error-default`. Replace it basded on that.

File: src/style.scss [hunk 2]
```diff
@@ -76,12 +79,12 @@ input {
...
-  max-height: calc(var(--auro-size-700, 56px) + 2px);
+  min-height: calc(var(--ds-size-700, 56px) + 2px);
+  max-height: calc(var(--ds-size-700, 56px) + 2px);
 
-  padding: var(--auro-size-400, 2rem);
-  padding-right: var(--auro-size-100, .5rem);
-  padding-bottom: var(--auro-size-100, .5rem);
+  padding: var(--ds-size-400, 2rem);
+  padding-right: var(--ds-size-100, .5rem);
+  padding-bottom: var(--ds-size-100, .5rem);
```
Comment (jason-capsule42) [inline src/style.scss:87]: All these fixed fallback values should be replaced with the equivelement scss variable.

File: src/style.scss [hunk 3]
```diff
@@ -76,12 +79,12 @@ input {
   position: relative;
   width: 100%;
 
-  min-height: calc(var(--auro-size-700, 56px) + 2px);
-  max-height: calc(var(--auro-size-700, 56px) + 2px);
+  min-height: calc(var(--ds-size-700, 56px) + 2px);
```
[the remark this answers is not in the document]
Comment (jason-capsule42) [reply src/style.scss:82]: Also all the hard coded "2px" values throughout the code should be `var(--ds-size-25)`.

File: src/style.scss [hunk 4]
```diff
@@ -155,13 +158,13 @@ input {
   padding: 0;
   padding-top: 1.8rem;
 
-  width: calc(100% - var(--auro-size-100, .5rem));
+  width: calc(100% - var(--ds-size-100, .5rem));
```
Comment (jason-capsule42) [inline src/style.scss:161]: `.5rem` should be `var(--ds-size-100)`.

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: src/style.scss (hunk 2 above)
Comment (fajar-apri-alaska) [reply src/style.scss:87]: sure, also will keep it as it is if there's none in the tokens.

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**